server/src/main/java/io/unitycatalog/server/persist/utils/ServerPropertiesUtils.java

```diff
@@ -28,20 +29,6 @@ private void loadProperties() {
     try (InputStream input = Files.newInputStream(Paths.get("etc/conf/server.properties"))) {
       properties.load(input);
       LOGGER.debug("Properties loaded successfully");
-      int i = 0;
-      while (true) {
```

Comment:
nit: personal preference but i like `for (int i = 0;; i++) ` loops when `break`-ing to exit.

Reply:
yeah this was just copy pasting what was already there, I can update to for loop